src/modules/reserve-overview/ReserveConfiguration.tsx

```diff
@@ -401,6 +402,25 @@ export const ReserveConfiguration: React.FC<{ reserve: ComputedReserveData }> =
                     <ReserveSubheader value={reserve.borrowCapUSD} />
                   </PanelItem>
                 )}
+                <PanelItem
+                  title={
+                    <ReserveFactorTooltip
+                      text={<Trans>Reserve factor</Trans>}
```

Comment:
Just looking at this and `TextWithTooltip.tsx` as a potential optimization.

We could eliminate having to pass in `text={<Trans>....</Trans>}` by just wrapping the text prop with the `Trans` component and be more DRY.  Doing so would allow any of these tooltip components to just take in `text="some text"` and it eventually get wrapped at the base level.

We'd update [this line](https://github.com/aave/interface/blob/main/src/components/TextWithTooltip.tsx#L27) to be `{text && <Typography {...rest}><Trans>{text}</Trans></Typography>}`

Thoughts?

Reply:
Nice catch @drewcook - i think its a nice little optmization

Reply:
Great question.  This is something worthy to test - make the change and run the extractor to see if it is indeed able to pick up on the dynamic values post-injection into the JSX templates.  I wonder if also running the extractor against the source files vs dist files after `yarn build` would make any difference to this scenario as well, just a side thought.  Of course we'll want it to run against our source files for daily development, though.  Regardless, I think it's worth testing and a great call out.

Reply:
@drewcook Could you create a separate issue to look in to this?

Reply:
I have been keeping an eye on this during development.  It looks like it _does not_ pick up on dynamic values.  It will stringify them with the JSX `{}` and an arbitrary variable for the value within it, rather than being able to compile it.  I believe this is because the i18n is manually run via compile/build time rather than runtime.  Those dynamic bits will be filled in with the proper values at runtime, and so the language interpreter wouldn't be able to guess those.

Here are some examples where dynamic bits are being used within JSX tags:

[one](https://github.com/aave/interface/blob/main/src/locales/de/messages.po#L105)
[two](https://github.com/aave/interface/blob/main/src/locales/fr/messages.po#L171)
[three](https://github.com/aave/interface/blob/main/src/locales/es/messages.po#L339)

Now, that said, the translation is able to translate anything else. But in the case where we'd want to pass in a large string/sentence as a prop and have it translated, I'm not sure the `.po` files will pick up on that, and at runtime, it may still be english, since that's what we wrote it in in code.

This said, I'm happy to put these findings into an issue though and fully test it if we think it's worth our while.

Reply:
New issue representing this - https://github.com/aave/interface/issues/960